CLAUDE.md

```
@@ -0,0 +1,59 @@
+node test/test-packet-completed.js             # Test packet.completed formatter only
+node test/test-packet-completed.js --simulate  # Test formatter without sending
+```
+
+## Architecture
+
+**Entry point:** `app.js` — Starts two servers:
+1. **Slack Bolt app** (Socket Mode) — maintains a WebSocket connection to Slack for message delivery
+2. **Express server** (HTTP) — receives incoming MCP webhooks on the configurable `MCP_WEBHOOK_URL_PATH`
+
+**Request flow:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)